code-challenges.js

```diff
@@ -13,52 +13,139 @@ var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
 
+//Creating a function that will take in a number and attempt to divide it by three and return a pleasant little string along with it. 
+const divisibleBy = (num) =>{
+    if(num / 3){
+        return `${num} is divisible by three!`
+    } else { return `${num} is not divisible by three  :( `
+    }
+}
+
+//Logging the function call three times, with the set variables above as arguments. I know there is a better way of doing this, but just wanted to do the long way
+console.log(divisibleBy(num1))
+console.log(divisibleBy(num2))
+console.log(divisibleBy(num3))
+*/
+
+
 
 
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
+/*
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+//Creating a function that will take in the array and spit out the array with only the first letter capitalized. 
+const capitalizedNouns = (nouns) => {
+    for (const nouns of randomNouns ){
+        console.log(nouns[0].toUpperCase() + nouns.substr(1))
+    }
+}
+
+//Calling the function to execute
+capitalizedNouns()
+
+
+*/
 
 
 
 
 // --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
 
+
+/*
+
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
 
+//Creation of filter and sorting function
+const numbersOnly = (arr) => {
+    return arr.filter(value => typeof value === "number").sort(function(a,b){return a-b})
+}
+
+// Calling the function to filter the array  and sort
+
+console.log(numbersOnly(mixedDataArray))
```

Comment:
woo! great use of `.filter` and `.sort`, as well as wrapping it all in one return. big improvement yay!